src/components/badges/BadgeWrapper.tsx

```diff
@@ -31,7 +33,8 @@ export default function ({
     <div
       className={classNamesToString(
         badgeWrapper(minted),
-        'last:odd:col-span-full'
+        'md:each-new-line:last:col-span-full',
+        'double-badge'
```

Comment:
What is `double-badge` supposed to mean? Is that supposed to be when there are two badges in a row? Is there maybe a more descriptive class name that we can use?

Reply:
Maybe `double-badge-last-row`?